FormidableLabs/react-native-app-auth
# Feat/issue #262 - allow to pass custom headers to oauth requests

Solve #262 
It is now possible to pass a `customHeaders` parameter to the `authorize` and `refresh` functions.
Thoses headers are injected in the connection used for the requests by the `CustomConnectionBuilder`

To test, in a react-native project:
* clone git@github.com:nihaux/react-native-app-auth.git somewhere in the android folder && checkout feat/issue-262
* setup react-native-app-auth but in `settings.gradle` replace the path in `project(':react-native-app-auth').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-app-auth/android')
` with the path to the cloned repo.
* in the react-native app, make a call to authorize with customHeaders set 
* in the debugger check for custom headers

- [x] include issue number that will be resolved by this PR (e.g. `Fixes #1234`)  
- [x] include a summary of the work
- [x] update tests (if applicable)
- [x] update readme (if applicable)
- [x] update typescript definitions (if applicable)
- [x] include steps to verify

TODO
- [ ] squash
